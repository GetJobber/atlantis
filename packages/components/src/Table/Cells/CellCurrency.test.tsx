import React from "react";
import renderer from "react-test-renderer";
import { CellCurrency } from "..";

it("renders a currency cell", () => {
  const tree = renderer.create(<CellCurrency value={42000000} />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders a currency cell in Pound Sterling", () => {
  const tree = renderer
    .create(<CellCurrency value={32074770} currency="GBP" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
