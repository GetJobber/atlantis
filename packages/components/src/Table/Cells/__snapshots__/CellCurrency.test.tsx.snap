// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a currency cell 1`] = `
<td
  className="cell right"
>
  <span
    className="numeric"
  >
    $42,000,000.00
  </span>
</td>
`;

exports[`renders a currency cell in Pound Sterling 1`] = `
<td
  className="cell right"
>
  <span
    className="numeric"
  >
    Â£32,074,770.00
  </span>
</td>
`;
